config/config.go

```diff
@@ -30,7 +30,7 @@ func Get() (*Config, error) {
 	if config.Debug {
 		config.PatternLibraryAssetsPath = "http://localhost:9002/dist/assets"
 	} else {
-		config.PatternLibraryAssetsPath = "//cdn.ons.gov.uk/dp-design-system/ba32e79"
+		config.PatternLibraryAssetsPath = "//cdn.ons.gov.uk/dp-design-system/afa6add"
```

Comment:
Bug fix for this will need to be applied in dp-design-system however a chat about how this exists should first be had and whether there is need for a fix.

Reply:
The fix will need to be in before this PR is merged 👌